bumble/controller.py

```diff
@@ -1000,12 +1000,16 @@ def on_hci_le_set_scan_parameters_command(self, command):
         '''
         See Bluetooth spec Vol 4, Part E - 7.8.10 LE Set Scan Parameters Command
         '''
-        self.le_scan_type = command.le_scan_type
-        self.le_scan_interval = command.le_scan_interval
-        self.le_scan_window = command.le_scan_window
-        self.le_scan_own_address_type = command.own_address_type
-        self.le_scanning_filter_policy = command.scanning_filter_policy
-        return bytes([HCI_SUCCESS])
+        ret = HCI_SUCCESS
```

Comment:
The "mutable ret" pattern is sometimes needed, but I think in this case it's not worth its cost as a potential bug source in future modifications. Can just return directly at the bottom of the `if`, then return unconditionally (no else needed).